old-stuff/index.html

```
@@ -0,0 +1,461 @@
+  const firebaseConfig = {
+    apiKey: "AIzaSyAyiiWVeRjxSsxUE8fR1DyCPEZwt7o3iHc",
+    authDomain: "plexstream-ef9b4.firebaseapp.com",
+    projectId: "plexstream-ef9b4",
+    storageBucket: "plexstream-ef9b4.firebasestorage.app",
+    messagingSenderId: "130967153299",
+    appId: "1:130967153299:web:e424baeb43541f1401bb66"
+  };
+  // Initialize Firebase
+  const app = initializeApp(firebaseConfig);
+</script>
+</script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)